vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events.cc

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events.h"
+
+#include <vector>
+
+#include "base/time/time.h"
+#include "bat/ads/internal/base/logging_util.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_event_info.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events_database_table.h"
+
+namespace ads {
+
+void LogTextEmbeddingHtmlEvent(
+    const ml::pipeline::TextEmbeddingInfo& text_embedding,
+    TextEmbeddingHtmlEventCallback callback) {
+  TextEmbeddingHtmlEventInfo text_embedding_html_event;
+  text_embedding_html_event.created_at = base::Time::Now();
+  text_embedding_html_event.locale = {};
```

Comment:
Is locale intentionally empty?

Reply:
Discussed and as not used we should remove

Reply:
or we should assign the resource locale

Reply:
Added